### DISCUSSION TITLE
I have waited 30 days for my benefits, is there something I'm missing?

### DISCUSSION THREAD

**Discussion by TheMipmap** [community/community#145550] [GitHub Education] [answered]
Hi,

I got approved for the educational program on the 25. october, but after 30 days I still haven't received acces to GitHub CoPilot, am i missing something?

Best regards,
Morten

**Comment by davevad93** [comment, marked as answer]
Hi @TheMipmap ,

What's the status of your application?

If the label is green and states "`Congrats! Your request for @TheMipmap was approved and your benefits will be available soon.` that means you have to wait. However It usually takes a few days or a couple a weeks at most. 

If the label is purple and states "`Congrats! Your request for @TheMipmap was approved and your benefits are now available.`", that means you should have access to your benefits. Try first going to the [pack offers page](https://education.github.com/pack), scroll down to the "***All offers***" section and select one of the available offers, for example [Copilot](https://education.github.com/pack/redeem/copilot-student), if you can redeem it you are fine, if you can't, I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.

For further reference, take a look at the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352):

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

And also the following discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287

**Reply by TheMipmap** [reply]
Thanks for the help, I just needed to redeem it :)

I did read the FAQ before asking this question. However, it didn't mention (at least that I found) that you have to redeem the benefits. It just said to raise a discussion on this forum instead of explaining the meaning of the purple/green labels. The pack offers page wasn't even mentioned. This could definitely be specified in a more clear manner...

Especially since I'm far from the only one having this problem.

**Reply by davevad93** [reply]
You're welcome @TheMipmap 👍. I agree, they should add it in the FAQs.
